2/TEXT/PL/ARIPCL02.TXT

```diff
@@ -0,0 +1,4 @@
+[0:00:000] – Kain zapewnił im zwycięstwo.
+[0:02:100]
+[0:03:000] - Oni pragnęli zguby Filarów, a on chętnie im się przysłużył.
```

Comment:
Usuń proszę ten dywiz `-` z początku linii, bo raz, że nie ma tam myślnika w wersji oryginalnej, a dwa, to jeśli już myślniki w dialogach powinny być zapisane półpauzą `–`. Przy foncie stałopozycyjnym w edytorze tekstu różnica może być subtelna i mało widoczna, ale już przy normalnym foncie różnica w długości tych znaków graficznych jest znacząca.
![image](https://github.com/user-attachments/assets/cc7c2f21-ba48-4c2c-bb50-f6a065fad49e)